README.md

```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
 * [Deployment / Helm](./docs/helm-deploy.md) - including how to roll back a deployment
 * [Runbook](./RUNBOOK.md)
 
+### Maintenance Banner
+* To display a maintenance banner to make users aware of any planned outages five environment variables are required. These are available for each environment in the helm_deploy folder -
+  * **SHOW_NOTIFICATION** - this must be set to true for the banner to manifest, by default it is false
+  * **NOTIFICATION_HEADER** - the header of the banner
+  * **NOTIFICATION_BODY** - the text in the body of the banner
+  * **START_DATE** - must be written in ISO format eg/YYYY-MM-DD, the banner will only be displayed **_after_** this date
```

Comment:
When this says "after" this date, does it mean that if I set it to 3 Sept, the first time the notification banner will be displayed is 4th September?

Reply:
It's inclusive, I'll update and add a link to the GDS document Beck signposted me to